olm-catalog/serverless-operator/project.yaml

```diff
@@ -31,4 +31,4 @@ dependencies:
   eventing_kafka: 1.1.0
   eventing_kafka_broker: 1.1.3
   cli: 1.1.0
-  operator: 1.0.1
+  operator: 1.2.0
```

Comment:
Since there is 1.2.2, should we use that one?
https://github.com/knative/operator/releases/tag/knative-v1.2.2

Reply:
Serving is on 1.2.0 but I guess should not matter?

Reply:
Yes, they can be 2 separate versions (in theory)

Reply:
Ok no problem let me do it.

Reply:
@pierDipi done

Reply:
+1 on using the latest version. we should never rely on all versions in the same `x.y.z` range, since they are independent (in theory)

Reply:
The patch version should not matter but it is a good thing in general to have versions close to each other. We dont want all versions to be on latest but have a very old component obviously.